app/services/archiving/move-claim-files-to-archive-file-store.js

```diff
@@ -0,0 +1,20 @@
+const Promise = require('bluebird')
+const config = require('../../../config')
+const mv = Promise.promisify(require('mv'))
+
+module.exports = function (archivedClaimData) {
+  var archiveEligibilityDocuments = archivedClaimData.DeleteEligibility
+  var reference = archivedClaimData.Claim.Reference
+  var eligibilityId = archivedClaimData.Claim.EligibilityId
+  var claimId = archivedClaimData.Claim.ClaimId
+
+  var directoryToCopy = `${config.FILE_UPLOAD_LOCATION}/${reference}-${eligibilityId}`
+  var targetDirectory = `${config.FILE_ARCHIVE_LOCATION}/${reference}-${eligibilityId}`
+
+  if (!archiveEligibilityDocuments) {
+    directoryToCopy = `${directoryToCopy}/${claimId}`
+    targetDirectory = `${targetDirectory}/${claimId}`
+  }
+
+  return mv(directoryToCopy, targetDirectory, {mkdirp: true})
```

Comment:
I got the following error when trying to archive a second claim after one claim had already been archived.

`error running task: IntSchema-323-ARCHIVE-CLAIM, error: Error: ENOTEMPTY: directory not empty, rename '../uploads/A3D2EEK-483' -> '../archive/A3D2EEK-483'`